what_the_fec/routes/contributions/endpoint_funcs.py

```diff
@@ -1,20 +1,116 @@
-from fastapi import Request
+from fastapi import Request, status
+from fastapi.responses import RedirectResponse
 from fastapi.templating import Jinja2Templates
-from sqlalchemy import Connection
+from sqlalchemy import Connection, text
 
-from what_the_fec.routes.helpers import generic_render_table
+from what_the_fec.routes.helpers import (
+    get_columns_information_dict,
+    get_columns_information_query,
+)
 
-TABLE_NAME = "contributions"
```

Comment:
what necessitates breaking the pattern with these?

1. TABLE_NAME + fstrings
2. get_all_func -> get_all

Reply:
I mimicked the `get_all` function in candidate_office_records because its's currently the only other entity that uses the dropdowns
https://github.com/nolandseigler/osu-cs340-project/blob/bba0bb8b006f5241732e54ab6a91af6c777c558b/what_the_fec/routes/candidate_office_records/endpoint_funcs.py#L12

Let me know if we should use the TABLE_NAME + fstrings and `get_all_func` in contributions and candidate_office_records instead